src/main/java/com/coveros/selenified/utilities/Property.java

```diff
@@ -153,13 +158,39 @@ public static boolean isProxySet() {
      *
      * @return String: the set proxy address, null if none are set
      */
-    public static String getProxy() {
+    public static String getProxy() throws InvalidProxyException {
         if (!isProxySet()) {
-            return null;
+            throw new InvalidProxyException("Proxy isn't set");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Proxy isn't set" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)